### DISCUSSION TITLE
Stopping using CHID.stop fails when fds file is concatenated as CHIT_cat.fds

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#13278] [Q&A]
Hi,

Stopping using CHID.stop fails when fds file is concatenated as CHID_cat.fds? Has this been noted before?

Regards!

**Comment by drjfloyd** [comment]
Use CHID_cat.stop.  When using &CATF, fds creates a new input file with the new CHID of CHID_cat and runs that file. We should add that to the guide as that is not very obvious.

**Reply by Ranhill** [reply]
Sorry, I did not write about that also. I used that CHID_cat.stop also. But it did not work!

**Reply by drjfloyd** [reply]
Make sure you have the CHID correct and are using the current release.  I ran test.fds (shown below) that concatenates the stuff.fds file (also shown below) with the file test_cat.stop present and got:

ERROR(109): Remove the file, test_cat.stop, from the current directory (CHID: test_cat)

ERROR: FDS was improperly set-up - FDS stopped (CHID: test_cat)


saved as test.fds
&HEAD CHID='test'/
&CATF OTHER_FILES='stuff.fds'/

saved as stuff.fds
&MESH XB=0,1,0,1,0,1,IJK=10,10,10/
&TIME T_END=1/
